### PR TITLE
hubert / Running husky prettier

### PR DESCRIPTION
**Pull request by hubert-deriv**
I added the husky dependency in a seperate PR, see for that this link: https://github.com/binary-com/deriv-developers-portal/pull/270

In this PR, I create a command with husky to automate and prettify developers code every time they make a commit.

In this first husky commit, husky will resolve all the current issues that are within the repo, so it will result in a lot of line changes. After this PR though, husky will always be there to keep our code more clean.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,14 +48,13 @@ Moreover, having these extensions will help you to speed up the development proc
...
 
-
 5.  **To start developing:**
 
     ```sh
     npm run dev
     ```
 
-1. Release to staging:
+6.  Release to staging:
```
Comment (matin-binary) [inline README.md:57]: Why does this change to `6`

Comment (hubert-deriv) [reply README.md:57]: I think readme was accidentally prettified somehow. Will revert the changes on the readme

**Review by matin-binary — APPROVED** [review]
LGTM

File: src/components/docs/Docs/MobilePageSelect/MobilePageSelect.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    };
+    return (
+        <select className={styles.mobilePageSelect} onChange={onSelectChange} value={currentPath}>
+            <option value='/docs/'>Quickstart</option>
+            <option value='/app-registration/'>App Registration</option>
+            <option value='/api-explorer/'>API Explorer</option>
+            <option value='/docs/resources/api-guide/'>API Guide</option>
+            <option value='/docs/resources/faq/'>FAQ</option>
+            <option value='/docs/resources/json-schemas/'>JSON Schemas</option>
+            <option value='/docs/resources/bug-bounty/'>Bug Bounty</option>
```
Comment (maryia-binary) [inline src/components/docs/Docs/MobilePageSelect/MobilePageSelect.jsx:22]:
```suggestion
            const links = [
                { value: '/docs/', text: 'Quickstart' },
                { value: '/app-registration/', text: 'App Registration' },
                { value: '/api-explorer/', text: 'API Explorer' },
                { value: '/docs/resources/api-guide/', text: 'API Guide' },
                { value: '/docs/resources/faq/', text: 'FAQ' },
                { value: '/docs/resources/json-schemas/', text: 'JSON Schemas' },
                { value: '/docs/resources/bug-bounty/', text: 'Bug Bounty' },
            ]
            // and the following inside <select>:
            { links.map(({ value, text }) => <option key={value} value={value}>{text}</option> }
```
@hubert-deriv what do you think about mapping these options? 😊

Comment (hubert-deriv) [reply src/components/docs/Docs/MobilePageSelect/MobilePageSelect.jsx:22]: Ah the mobile page select! Well, that is strange, we removed the mobile page select actually, as it became redundant. Looks like it has been accidentally added again. I will remove it again.

**Review by mahdiyeh-fs — APPROVED** [review]

**Review by maryia-binary — APPROVED** [review]

**Pull request merged**